### ISSUE TITLE
AltStore stays loading when signing in Apple ID

### ISSUE THREAD

**Issue by quiqueenri** [rileytestut/AltStore#810]
AltStore(Windows Version)
When attempting to sign in with Apple Id*, the Sign In Button will infinitely loop after inputting Apple Id and Password, never logging in, making the app unable to sign in and to install apps. The App does NOT crash when doing this.

*iPad iOS 14.7.1 Connected to PC, On Same Wifi

**Comment by zillydev** [comment]
Same issue with me. It didn't use to happen before.

*iPhone SE (2020) iOS 16.0.2

**Comment by shanegillio** [comment]
hello, sorry for the lack of response - is the issue still occurring on the latest version?

**Comment by wimoy1** [comment]
yes, the problem remains, I have an infinite loading at the entrance (Iphone 11 IOS 16.1.2)
